### PR TITLE
EDU-1041: Update slugs for Cloud information and account-setup pages

### PR DESCRIPTION
**Pull request by djmagee**
## Previews

**https://temporal-documentation-git-edu-1041-cloud-slugs.preview.thundergun.io/cloud/introduction**
**https://temporal-documentation-git-edu-1041-cloud-slugs.preview.thundergun.io/cloud/account-setup/**
**https://temporal-documentation-git-edu-1041-cloud-slugs.preview.thundergun.io/cloud/saml**
**https://temporal-documentation-git-edu-1041-cloud-slugs.preview.thundergun.io/cloud/audit-logging**

## What does this PR do?

- Changes slugs for the following nine [previously seven] pages:
  - `/cloud/introduction/security-cloud` → `/cloud/security`
  - `/cloud/introduction/operating-envelope` → `/cloud/operating-envelope`
  - `/cloud/introduction/pricing` → `/cloud/pricing`
  - `/cloud/introduction/support` →`/cloud/support`
  - `/cloud/account-setup/certificates` → `/cloud/certificates`
  - `/cloud/account-setup/namespaces` → `/cloud/namespaces`
  - `/cloud/account-setup/users` → `/cloud/users`
  - `/cloud/how-to-manage-saml-with-temporal-cloud` → `/cloud/saml`
  - `/cloud/how-to-manage-audit-logging` → `/cloud/audit-logging`
- Updates current links to these pages.
- Adds redirects.

### CODE REVIEW

File: docs-src/cloud/cloud-intro.md [hunk 1]
```diff
@@ -14,6 +14,7 @@ To request a Temporal Cloud account, complete the [request form](https://pages.t
 :::
 
 - [Introduction to Temporal Cloud](/cloud/introduction)
+  - [Security](/cloud/security)
```
Comment (flossypurse) [inline docs-src/cloud/cloud-intro.md:17]: link to the node id of the first node in the guide - and link magic will do this 
Plus we can be sure the links are rewritten for guide indexing - so urls in result matches actually work.

File: docs-src/cloud/namespace-setup-considerations.md [hunk 2]
```diff
@@ -17,7 +17,7 @@ Customers should consider not only a Namespace naming convention but also how to
 Before considering an appropriate Namespace configuration, you should be aware of the following constraints:
 
 - Each Temporal account has a default limit of 10 Namespaces.
-  You can request an increase by [creating a ticket for Temporal Support](/cloud/introduction/support#support-ticket).
+  You can request an increase by [creating a ticket for Temporal Support](/cloud/support#support-ticket).
```
Comment (flossypurse) [inline docs-src/cloud/namespace-setup-considerations.md:20]: this is a docs-src node - link to the id of the node - NOT the final build slug.

File: docs-src/cloud/namespace-setup-considerations.md [hunk 3]
```diff
@@ -28,7 +28,7 @@ Before considering an appropriate Namespace configuration, you should be aware o
   Access to Temporal by [Worker Processes](/workers#worker-process) is permitted at the Namespace level.
   Isolating applications or environments (development, test, staging, production) should take this into consideration.
 - A Namespace is an endpoint.
-  To access a Namespace from a Temporal Client requires mTLS authorization, which requires [CA certificates](/cloud/account-setup/certificates#ca-certificates).
+  To access a Namespace from a Temporal Client requires mTLS authorization, which requires [CA certificates](/cloud/certificates#ca-certificates).
```
Comment (flossypurse) [inline docs-src/cloud/namespace-setup-considerations.md:31]: This needs to be updated too - link to the id of the node.

File: docs-src/cloud/cloud-intro.md [hunk 4]
```diff
@@ -23,6 +24,6 @@ To request a Temporal Cloud account, complete the [request form](https://pages.t
   - [Manage certificates](/cloud/certificates-intro)
   - [Manage users](/cloud/users-intro)
 - [Metrics setup](/cloud/metrics-intro)
-- [SAML](/cloud/saml-intro)
+- [SAML](/cloud/saml)
 - [Audit logging](/cloud/what-is-audit-logging)
 - [tcld (Temporal Cloud command-line interface)](/cloud/tcld)
```
Comment (flossypurse) [inline docs-src/cloud/cloud-intro.md:29]: here too

File: docs-src/cloud/security-cloud-platform.md [hunk 5]
```diff
@@ -33,13 +33,13 @@ For more information, see [Requirements for CA certificates in Temporal Cloud](/
 
 Authentication to gRPC endpoints is provided by mTLS per Namespace.
 
-For more information, see [How to manage SAML authentication with Temporal Cloud](/cloud/how-to-manage-saml-with-temporal-cloud).
+For more information, see [How to manage SAML authentication with Temporal Cloud](/cloud/saml).
```
Comment (flossypurse) [inline docs-src/cloud/security-cloud-platform.md:36]: link to the node.

File: docs-src/cloud/security-cloud-platform.md [hunk 6]
```diff
@@ -33,13 +33,13 @@ For more information, see [Requirements for CA certificates in Temporal Cloud](/
...
 
 ### Access
 
 Authorization is managed at the account and Namespace level.
 Users and systems are assigned one or more preconfigured roles.
-Users hold [account-level Roles](/cloud/what-are-the-account-level-roles-for-users-in-temporal-cloud) of administrators, developers, and read-only users.
+Users hold [account-level Roles](/cloud/users#account-level-roles) of administrators, developers, and read-only users.
```
Comment (flossypurse) [inline docs-src/cloud/security-cloud-platform.md:42]: link to the node.

File: docs-src/concepts/what-is-the-temporal-web-ui.md [hunk 7]
```diff
@@ -116,7 +116,7 @@ Click a Schedule to see details, such as configured frequency, start and end tim
 
 On Temporal Cloud, **Settings** is visible only to [Global Admins](/cloud/#account-level-roles).
 
-Click **Settings** to see and manage the list of users in your account and to set up integrations such as [Observability](/cloud/metrics-intro#configure-a-metrics-endpoint-using-temporal-cloud-ui) and [Audit logging](/cloud/how-to-manage-audit-logging).
+Click **Settings** to see and manage the list of users in your account and to set up integrations such as [Observability](/cloud/metrics-intro#configure-a-metrics-endpoint-using-temporal-cloud-ui) and [Audit logging](/cloud/audit-logging).
```
Comment (flossypurse) [inline docs-src/concepts/what-is-the-temporal-web-ui.md:119]: second link - link to the node

**Review by flossypurse — CHANGES_REQUESTED** [review]
1. link to the node ids so that links are re-written based on the build context (either as a site or for search indexing)
2. In this PR can you also adjust the .gitattributes file so that all generated cloud guides are not shown by default in the PRs?

File: docs-src/cloud/cloud-intro.md (hunk 4 above)
Comment (djmagee) [reply docs-src/cloud/cloud-intro.md:29]: The id here is `index` for both the config file and the first node, so I think this one is GTG.

**Review by flossypurse — APPROVED** [review]
looking good.

**Pull request merged**